activesupport/lib/active_support/callbacks.rb

```diff
@@ -80,8 +80,12 @@ module Callbacks
     #     save
     #   end
     def run_callbacks(kind, &block)
-      callbacks = send("_#{kind}_callbacks")
+      send "_run_#{kind}_callbacks", &block
+    end
+
+    private
 
+    def __run_callbacks__(callbacks, &block)
```

Comment:
Renamed to `__run_callbacks__` from `_run_callbacks` to avoid breaking https://github.com/rails/rails/issues/19405.